<launch>

    <!--<node pkg="self_aiming" name="server_node" type="server_node.py" output="screen"/>-->
    <!--<node pkg="rqt_reconfigure" name="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>-->

    <node name="feedforward_ctl" pkg="gimbal_controller" type="visual_servo_controller_with_feedforward" output="screen">
        <param name="Kp" type="double" value="-20.0"/>
        <param name="Kd" type="double" value="0.0"/>
        <param name="publisher_topic" type="string" value="/cmd_vel"/>
        <param name="omega_pub_topic" type="string" value="/visual_estimate"/>
        <param name="omega_topic" type="string" value="/can_transimit/omega_cam"/>
        <param name="cfg_file_name" type="string" value="/home/ros/ws/src/6_controller/gimbal_controller/cfg/camera_tracking_camera_calib.yaml"/>
    </node>
</launch>
